components/atoms/Select/select.tsx

```diff
@@ -19,7 +19,7 @@ const SelectTrigger = React.forwardRef<
   <SelectPrimitive.Trigger
     ref={ref}
     className={clsx(
-      "flex h-[2.1rem] w-full items-center justify-between rounded-lg overflow-hidden   px-3 py-2 text-sm border  focus:outline-none  disabled:cursor-not-allowed disabled:opacity-50",
+      "flex min-h-[2.1rem] w-full items-center justify-between rounded-lg overflow-hidden  px-3 py-[5.8px] text-sm border  focus:outline-none  disabled:cursor-not-allowed disabled:opacity-50",
```

Comment:
Hardcoding pixels `py-[5.8px]` fixes this page, but I am curious if it breaks others.

Reply:
Yes, I am also skeptical about that. I made sure the height remains the same if the input value is not making the whole Select component occupy two lines or more; or if its empty. 
I have searched in the application and checked one other place where Select component was being used and compared it with the older implementation and that was working fine as well. Talking about the settings page.
If anyone can point me to places where it is being used I can check it myself and add some pictures so we can confirm it.